### PR TITLE
allow brackets and apostrophies and macrons on vowels

### PR DESCRIPTION
**Pull request by JacobBriggsAckama**
Allow brackets and apostrophies and macrons on vowels in the term search, so that "change (clo" and "can't" can be searched for and have results returned.

Note that the autocomplete search seems to be restricted on the freelex side, so "can't" and "change (clo" will not autocomplete to anything - but that hasn't changed.

### CODE REVIEW

**Comment by JacobBriggsAckama** [comment]
seems like the new regex that sanitises search terms isn't allowing hyphens anymore for some reason

**Comment by JacobBriggsAckama** [comment]
AND fixed, hyphens are allowed

File: app/services/search_query_sanitization_service.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class SearchQuerySanitizationService
   ANYTHING_EXCEPT_NUMBERS_AND_SPACE_REGEX = /[^[[:digit:]] ]+/.freeze
   ANYTHING_EXCEPT_NUMBERS_SPACE_PERIOD_REGEX = /[^[[:digit:]]. ]+/.freeze
   ANYTHING_EXCEPT_NUMBERS_REGEX = /[^[[:digit:]]]+/.freeze
-  ANYTHING_EXCEPT_LETTERS_NUMBERS_HYPHEN_SPACE_REGEX = /[^[[:alnum:]] -]+/.freeze
+  ANYTHING_EXCEPT_LETTERS_NUMBERS_COMMON_PUNCTUATION_REGEX = /[^[[:alnum:]] ā ē ī ō ū Ā Ē Ī Ō Ū \- () ']+/.freeze
```
Comment (eoinkelly) [inline app/services/search_query_sanitization_service.rb]: I think you probably only need one space in this character group?

Comment (JacobBriggsAckama) [reply app/services/search_query_sanitization_service.rb]: oh true why did I do it like that for?

Comment (G-Rath) [reply app/services/search_query_sanitization_service.rb]:
```suggestion
  ANYTHING_EXCEPT_LETTERS_NUMBERS_COMMON_PUNCTUATION_REGEX = /[^[[:alnum:]]āēīōūĀĒĪŌŪ\-() ']+/.freeze
```

**Review by G-Rath — APPROVED** [review]

**Pull request merged**